.github/workflows/unit_tests.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        image: ["postgres:10", "postgres:11", "postgres:16"]
+        image: ["postgres:12", "postgres:15", "postgres:16"]
```

Comment:
Should be postgres:13 instead of postgres:12.

https://github.com/cloudfoundry/cf-deployment/pull/1237